bigbluebutton-html5/imports/ui/components/audio/captions/speech/service.js

```diff
@@ -0,0 +1,171 @@
+import _ from 'lodash';
+import { diff } from 'bbb-diff';
```

Comment:
```suggestion
import { diff } from '@mconf/bbb-diff';
```

Reply:
Hmmm, noticed this is on 2.5 and not 2.6 yet. Maybe just be ready for the impact.